### ISSUE TITLE
Ocropy models only working with full path

### ISSUE THREAD

**Issue by SB2020-eye** [OCR-D/ocrd_all#241]
Hi. This is just fyi, since I eventually got my scripts to work.

I've been trying out `ocrd-cis-ocropy-recognize` for the first time. Unlike my experience with the `sbb` and `tesserocr` tools and models, the name for model wouldn't work for me with `ocropy`. Eventually I figured to try the full path, and it worked.

(So, eg,

- `ocrd resmgr list-installed` lists `LatinHist.pyrnn.gz`

- `ocrd-cis-ocropy-recognize -I OCR-D-SEG19 -O OCR-D-SEG26 -P textequiv_level glyph -P model LatinHist.pyrnn.gz` returns
```  File "/home/scott/src/github/OCR-D/ocrd_all/venv/lib/python3.6/site-packages/ocrd_cis/ocropy/ocrolib/common.py", line 653, in ocropus_find_file
    raise FileNotFound(fname)
ocrd_cis.ocropy.ocrolib.exceptions.FileNotFound: file not found LatinHist.pyrnn.gz
```

- `ocrd-cis-ocropy-recognize -I OCR-D-SEG19 -O OCR-D-SEG26 -P textequiv_level glyph -P model ~/.local/share/ocrd-resources/ocrd-cis-ocropy-recognize/LatinHist.pyrnn.gz` works)

**Comment by bertsky** [comment]
Thanks for your report!

Note that there's already a solution for this in the line [here](https://github.com/cisocrgroup/ocrd_cis/pull/83), but before we can adapt ocrd_cis to the new resmgr I'll have to clear away some unfinished work in the repo.

In the meantime, besides absolute paths, the relative paths still work under the [old resolution scheme](https://github.com/cisocrgroup/ocrd_cis/issues/41#issuecomment-608348012).

(We could transfer the issue to ocrd_cis, but since we already have the PR there, let's keep it here until we can merge ocrd_cis' fixed `master` again.)

**Comment by SB2020-eye** [comment]
@bertsky , I'm not in a rush. But if you think of it, it would be great if you could report back here when this happens. (And I'll additionally know to close the issue.) Thanks.

**Comment by bertsky** [comment]
Fixed by #362
